blog/entries/models.py

```diff
@@ -2,11 +2,18 @@
 
 from django.contrib.auth.models import User
 
+class Tag(models.Model):
+	tag_text = models.CharField(max_length=25)
+
+	def __str__(self):
+		return self.tag_text
+
 class Entry(models.Model):
 	entry_title = models.CharField(max_length=50)
 	entry_text = models.TextField()
 	entry_date = models.DateTimeField(auto_now_add=True)
 	entry_author = models.ForeignKey(User, on_delete=models.CASCADE)
+	entry_tags = models.ManyToManyField(Tag) # why did Tag need to be a string, or defined before this?
```

Comment:
in fact, this is not the case. the string just made it break, you have to define the model BEFORE you use it in a statement, hence Tag model has been moved to the top.